lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -23,22 +23,26 @@ local function restrictPropCoreFunctions()
     }
 
     for _, signature in pairs( restrictedFunctions ) do
-        local oldFunc = wire_expression2_funcs[signature][3]
+        if wire_expression2_funcs then
```

Comment:
CECO? (`continue` early `continue` often)

Reply:
sure, if you introduce the `continue` keyword into the language for me, boss

Reply:
Well, do I have a surprise for you..

```lua
local a = {"this", "has", "many", "entries"}

for k, v in pairs(a) do
    print(v)
end
```
results in:
```
this
has
many
entries
```

however, this:

```lua
local a = {"this", "has", "many", "entries"}

for k, v in pairs(a) do
    if v ~= "this" then continue end
	
    print(v)
end
```
results in
```
this
```